contracts/orderbook/OrderBookMarketOrders.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity ^0.8.4;
+import "./OrderBookEvents.sol";
+import "./OrderBookStorage.sol";
+
+
+contract OrderBookMarketOrders is OrderBookEvents,OrderBookStorage,safeTransfers{

```

Comment:
is there a way to use
import "@openzeppelin-3.4.0/token/ERC20/SafeERC20.sol";
using SafeERC20 for IERC20;